assets/src/edit-story/components/panels/index.js

```diff
@@ -90,15 +90,13 @@ export function getPanels(elements, options = {}) {
   }
 
   const isBackground = elements.length === 1 && elements[0].isBackground;
+  const isBackgroundMedia = isBackground && !elements[0].isDefaultBackground;
```

Comment:
This could probably be moved to just before usage and then the addition of `isBackground &&` appears irrelevant.